### ISSUE TITLE
General Show Feedback

### ISSUE THREAD

**Issue by garricn** [SwiftCommunityPodcast/podcast#3]

**Comment by GlobeFlex** [comment]
Thank you so much for this episode.  I would love to suggest a topic.  I would love to hear a topic on learning Swift from scratch or as in my case switching from VB.Net / C# / Python 3 which I use at work.  I feel that starting off right is important and could be a great topic.  

The background of Swift was so great to hear from Chris.  I would love to be able to help out the Swift community.  I have tried from day one of the first Swift 1 playground to really dive into this language.  I need to make or get involved in a project to make it all stick with me.

Thank you Chris, Garric, and John

**Comment by artrmz** [comment]
Since it's "for the Swift Community" it would be cool to help unpopular developers get some attention, I suggest adding a short section (1-3 minutes? or just 1 mention per episode) in each episode and promote some random (or maybe better some unpopular developer you think deserve some attention) to promote their blog, apps, open source projects etc.

Add an intro jingle like in John's own podcast. Things are getting excited when you hear that.

**Comment by Roslund** [comment]
I agree with @artrmz, it would be great if there were some shoutouts to open source projects.

**Comment by lattner** [comment]
I think shoutouts are a great idea.  How do you suggest we organize this in prep for the next show?

**Comment by artrmz** [comment]
@lattner Open an issue (to not spam with issues for each proposal) where people can suggest who should be promoted and can vote 👍👎 for certain shoutouts (but what to do with those that has been already promoted? mark them with a ❤ reaction or so?).

Then the host (an maybe each guest?) of the show picks one either from the issue here or can just a shoutout to anyone chosen by himself/herself.

**Comment by dndydon** [comment]
Great concept for community podcast with GitHub organization backing it. Thank you for founding it.  

I consume a lot of Swift podcasts, meetups,  and tutorials, as well as many other “shows.”  Your heartfelt stories and intelligent commentary based on respect for beginners is refreshing and encouraging. No surprise there, as the larger Swift community (and all Apple tech) has always had this culture of inclusion. This is a great start.

**Comment by garricn** [comment]
> Since it's "for the Swift Community" it would be cool to help unpopular developers get some attention, I suggest adding a short section (1-3 minutes? or just 1 mention per episode) in each episode and promote some random (or maybe better some unpopular developer you think deserve some attention) to promote their blog, apps, open source projects etc.
> 
> Add an intro and outro jingles like in John's own podcast. Things are getting excited when you hear that. Especially outro since it ended so suddenly.

Great ideas! How can we find those developers? Do you have any to suggest?

**Comment by artrmz** [comment]
@garricn Check my post from above (https://github.com/SwiftCommunityPodcast/podcast/issues/3#issuecomment-455455131).

Either people should suggest themselves/some devs they like or hosts/guests can choose whoever they want to.

**Comment by Konrad77** [comment]
Love the first episode. Havent had time to listen to the new episode yet. 
The feedback so far:

1. Chris should get a better microphone. 
2. Whats the best practise structuring large projects in Swift. Apples sample code is rather small.

**Comment by tommyang** [comment]
Thanks for the show. I would love to see [chapters](https://9to5mac.com/2018/06/19/ios-12-how-to-use-podcasts-chapters/) support added so that it is easier to navigate.
There are tools such as [Forecast](https://overcast.fm/forecast) to help with this.

**Comment by 8bitmp3** [comment]
Awesome show. Looking forward to the upcoming episode. 
Suggestion - invite @dan-zheng to talk about _differentiable Swift_
(cc @saeta @lattner @ematejska) 

www.github.com/tensorflow/swift
www.github.com/tensorflow/swift-apis
www.github.com/tensorflow/swift-models

![image](https://user-images.githubusercontent.com/19637339/75629362-5d536a00-5bd9-11ea-91c7-ffc74ba52257.png)
